shop/src/main/java/ro/msg/learning/shop/repository/CustomerRepository.java

```diff
@@ -0,0 +1,10 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.domain.Customer;
+import java.util.UUID;
+
+@Repository
```

Comment:
`@Repository`  is an optional annotation. You can remove it from all repository interfaces and Spring will still know the they are repositories since they extend `JpaRepository`.